packages/design-system-docs/src/pages/components/ChoiceList/Choice.example.jsx

```diff
@@ -1,155 +1,66 @@
-import { Choice, Dropdown } from '@cmsgov/design-system';
+import { Choice, TextField } from '@cmsgov/design-system';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const dropdownOptions = [
-  { label: 'A', value: 'A' },
-  { label: 'B', value: 'B' },
-  { label: 'C', value: 'C' },
-  { label: 'D', value: 'D' },
-  { label: 'E', value: 'E' },
-  { label: 'F', value: 'F' },
-  { label: 'G', value: 'G' },
-  { label: 'H', value: 'H' },
-];
-
-const childDropdown = (
-  <Dropdown
-    options={dropdownOptions}
-    defaultValue="B"
-    label="Dropdown example"
-    labelClassName="ds-c-label ds-u-margin-top--0"
-    name="dropdown_choices_field"
-  />
+const childField = (
+  <TextField label="Child field" labelClassName="ds-u-margin-top--0" name="textfield_child" />
 );
 
 ReactDOM.render(
-  <div>
-    <fieldset className="ds-c-fieldset">
-      <legend className="ds-c-label">
-        <span>Checkboxes</span>
-      </legend>
-      <Choice
-        defaultChecked
-        hint="Checkbox A hint"
-        name="checkbox_choice"
-        type="checkbox"
-        value="a"
-      >
-        Checkbox A
-      </Choice>
-      <Choice name="checkbox_choice" type="checkbox" value="b">
-        Checkbox B
-      </Choice>
-      <Choice name="checkbox_choice" type="checkbox" value="c">
-        Checkbox C
-      </Choice>
-    </fieldset>
-
-    <fieldset className="ds-c-fieldset">
-      <legend className="ds-c-label">
-        <span>Checkboxes with children</span>
-      </legend>
-      <Choice hint="Checkbox A hint" name="checkbox_choice_children" type="checkbox" value="a">
-        Checkbox A
-      </Choice>
-      <Choice
-        defaultChecked
-        name="checkbox_choice_children"
-        type="checkbox"
-        value="b"
-        checkedChildren={<div className="ds-c-choice__checkedChild">{childDropdown}</div>}
-      >
-        Checkbox B - with children
-      </Choice>
-      <Choice name="checkbox_choice_children" type="checkbox" value="c">
-        Checkbox C
-      </Choice>
-    </fieldset>
-
-    <hr className="ds-u-margin-top--2" />
-
-    <fieldset className="ds-c-fieldset">
-      <legend className="ds-c-label">
-        <span>Radio buttons</span>
-      </legend>
-      <Choice defaultChecked name="radio_choice" type="radio" value="a">
```

Comment:
Consider adding a shared `name=` to the radio choices so the user can toggle them on and off to see how they work. Otherwise, you can't uncheck a radio button once you check it.

Reply:
I'm thinking that it may be good to include `fieldsets` in the `choice` examples to highlight that if we use `choiceList`, the fieldsets/legends are taken cared off by the react `choiceList` component. On the other hand, if we use the `choice` component, we need to manage all these html element separately.

Reply:
But, you're right, people will write custom code anyway, so it's also good to go without the `fieldsets` info.

Reply:
I ended up adding fieldsets back @nichia